server/src/app/models/User.js

```diff
@@ -0,0 +1,38 @@
+import mongoose from "mongoose";
+
+const UserSchema = mongoose.Schema(
+  {
+    userId: {
+      type: Number
+    },
+    name: {
+      type: String,
+      required: true
+    },
+    email: {
+      type: String,
+      unique: true,
+      required: true,
+      trim: true,
+      lowercase: true,
+      match: /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/,
+      validate(value) {
+        // validation here
```

Comment:
you can add the validation here instead of passing out the comment. Try to complete the PR, instead of adding as TODO or leaving as comment.

Reply:
Done